Makefile

```diff
@@ -6,157 +6,22 @@ XARGS ?= xargs -I {}
-	@ echo "[$@]: Terraform files PASSED lint test!"
-
-json/%: FIND_JSON := find . $(FIND_EXCLUDES) -name '*.json' -type f
-json/lint: | guard/program/jq
-	@ echo "[$@]: Linting JSON files..."
-	$(FIND_JSON) | $(XARGS) bash -c 'cmp {} <(jq --indent 4 -S . {}) || (echo "[{}]: Failed JSON Lint Test"; exit 1)'
-	@ echo "[$@]: JSON files PASSED lint test!"
-
-json/format: | guard/program/jq
-	@ echo "[$@]: Formatting JSON files..."
-	$(FIND_JSON) | $(XARGS) bash -c 'echo "$$(jq --indent 4 -S . "{}")" > "{}"'
-	@ echo "[$@]: Successfully formatted JSON files!"
-
-sh/%: FIND_SH := find . $(FIND_EXCLUDES) -name '*.sh' -type f -print0
-sh/lint: | guard/program/shellcheck
-	@ echo "[$@]: Linting shell scripts..."
-	$(FIND_SH) | $(XARGS) shellcheck {}
-	@ echo "[$@]: Shell scripts PASSED lint test!"
-
-yaml/lint: | guard/program/yamllint
-	yamllint --strict .
-
 cfn/%: FIND_CFN ?= find . $(FIND_EXCLUDES) -name '*.template.cfn.*' -type f
-cfn/lint: | guard/program/cfn-lint
-	$(FIND_CFN) | $(XARGS) cfn-lint -t {}
-
+cfn/version: guard/program/yq
 cfn/version:
 	$(FIND_CFN) | $(XARGS) bash -c "yq -e '.Metadata.Version | test(\"^$(VERSION)$$\")' {} > /dev/null || (echo '[{}]: BAD/MISSING Cfn Version Metadata'; exit 1)"
 
-eclint/lint: | guard/program/eclint
-	@ echo "[$@]: Linting all files with eclint..."
-	eclint check
-	@ echo "[$@]: All files PASSED lint test!"
-
-docs/%: FOUND_DIR ?= find ${@F} -type d \( -name "lx*" -o -name "win*" \)
-docs/%: README_DIR ?= .
-docs/%: README_BASE ?= _docs/MAIN.md.jinja
-docs/%: README_MAIN := $(README_DIR)/MAIN.md
-docs/%: README_PARTS ?= $(README_MAIN) <(echo) <($(BIN_DIR)/terraform-docs.sh markdown $(README_DIR))
-docs/%: README_FILE ?= $(README_DIR)/README.md
-
-docs/template: | guard/program/jinja2
-	@ echo "[$@/$(README_DIR)]: Templating doc..."
-	jinja2 -D name=$(README_DIR) $(README_BASE) > $(README_MAIN)
-
-docs/lint: | guard/program/terraform-docs
-	@ echo "[$@/$(README_DIR)]: Linting documentation file.."
-	diff $(README_FILE) <(cat $(README_PARTS))
-	@ echo "[$@/$(README_DIR)]: Documentation file PASSED lint test!"
-
-docs/generate: | guard/program/terraform-docs
-	@ echo "[$@/$(README_DIR)]: Creating documentation file.."
-	cat $(README_PARTS) > $(README_FILE)
-	@ echo "[$@/$(README_DIR)]: Completed successfully!"
-
-docs/template/%: | %
-	@ echo "[$@]: Templating documentation main file.."
-	$(FOUND_DIR) | $(XARGS) $(MAKE) docs/template README_DIR={}
-	@ echo "[$@]: Completed successfully!"
-
-docs/lint/%: | docs/template/% tfdocs-awk/install
-	@ echo "[$@]: Starting documentation linting for $(@F).."
-	$(FOUND_DIR) | $(XARGS) $(MAKE) docs/lint README_DIR={}
-	@ echo "[$@]: Linting documents for $(@F) complete!"
-
-docs/generate/%: | docs/template/% tfdocs-awk/install
-	@ echo "[$@]: Start generating documentations for $(@F).."
-	$(FOUND_DIR) | $(XARGS) $(MAKE) docs/generate README_DIR={}
-	@ echo "[$@]: Generate documentations for $(@F) complete!"
+cfn/lint: | guard/program/cfn-lint
```

Comment:
Looks like this was deleted and re-added? But this target is included in tardigrade-ci, isn't it?

Reply:
I removed it initially in favor of the tardigrade-ci make target but added it back to avoid breaking the gitlab-ci config